### PR TITLE
Update setup-and-auto-renew-ssl-certificates.md

### PR DESCRIPTION
**Pull request by blessedmadukoma**

### CODE REVIEW

File: src/data/guides/setup-and-auto-renew-ssl-certificates.md [hunk 1]
```diff
@@ -59,6 +59,25 @@ You can either run the following command for the interactive mode where you will
...
+
+```shell
+sudo apt-get update
+sudo apt-get install python3-certbot-apache 
+```
+
+Then re-run:
+```shell
+sudo certbot --apache
+```
```
Comment (rtk-rnjn) [inline src/data/guides/setup-and-auto-renew-ssl-certificates.md:78]: Fenced code blocks should be surrounded by blank lines.

File: src/data/guides/setup-and-auto-renew-ssl-certificates.md [hunk 2]
```diff
@@ -59,6 +59,25 @@ You can either run the following command for the interactive mode where you will
...
+
+
```
Comment (rtk-rnjn) [inline src/data/guides/setup-and-auto-renew-ssl-certificates.md:80]: Multiple consecutive blank lines

**Review by blessedmadukoma — COMMENTED** [review]
I have implemented the changes

**Comment by blessedmadukoma** [comment]
I am standing by for your feedback

**Comment by kamranahmedse** [comment]
Thank you for the PR but I don't think there is any need to include this debugging information.

**Comment by blessedmadukoma** [comment]
> Thank you for the PR but I don't think there is any need to include this debugging information.

I tried running the original command, which didn't work even though I had my python3 alias as python. The certbot command was specific to CentOS to use python3-certbot, that's why I opened a PR. Thank you

**Pull request closed without merging**